turbo.json

```diff
@@ -5,10 +5,10 @@
     "build": {
       "dependsOn": ["^build"],
       "inputs": ["src/**/*", "package.json"],
-      "outputs": ["dist-types/**", "dist-cjs/**", "dist-es/**"]
+      "outputs": ["dist-types/**/*", "dist-cjs/**/*", "dist-es/**/*"]
     },
     "test": {
-      "dependsOn": ["build"],
+      "dependsOn": [],
```

Comment:
Both karma and jest don't need the package in question to be built to run unit tests. 

jest (via babel) and karma (via typescript) compile the unit test sources during the test run.